public/app/features/panel/components/VizTypePicker/PanelTypeCard.tsx

```diff
@@ -116,7 +116,7 @@ const getStyles = (theme: GrafanaTheme2) => {
       background: ${theme.colors.action.selected};
     `,
     disabled: css`
-      opacity: ${theme.colors.action.disabledOpacity};
+      opacity: 0.6;
```

Comment:
🤔 is it possible to create a new theme variable with the right opacity or change the original one? Maybe we can fix other places that experience the same problem 😄

Reply:
You have a good point :), I think for now we can keep it like this